docs/maven-tutorial.md

```diff
@@ -0,0 +1,146 @@
+# OpenShift + Maven + Nexus Tutorial
```

Comment:
openshift is implied by this being in the openshift project.  "Maven + Nexus Mirror Tutorial"  (don't love that either, but it'll work for now)